src/blocks/product_card.css

```diff
@@ -0,0 +1,88 @@
+.product-card {
+  width: 200px;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 6px;
+  padding: 32px 16px 16px;
+}
+
+.product-card__img {
+  display: block;
+  background-image: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: 160px 134px;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto;
+}
+
+.product-card__name {
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+  margin-top: 40px;
+  font-size: 12px;
+}
+
+.product-card__code {
+  color: #616070;
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.product-card__rate {
+  margin-top: 17px;
```

Comment:
I don't think it was 17 on the mockup

Reply:
It`s  16px on the mockup, but I can`t pass tests with that value, 17 made as it should be. But I`m still searching for the mistake...

Reply:
Ok, found